tests/js/spec/components/events/eventCause.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
```

Comment:
I converted this test to tsx as I was debugging it...  It's nice to have TS support for mocks and debugging

Reply:
Would be nice to split this out into it's own PR so we can merge sooner than later :)

Reply:
I needed to fix the test, I can switch it back to jsx if you think it'll make the test easier to review.

Reply:
Switched back to jsx